package.json

```diff
@@ -76,6 +78,9 @@
     "release-it-lerna-changelog": "^2.3.0",
     "webpack": "^5.58.2"
   },
+  "peerDependencies": {
+    "ember-source": "^3.8 || 4.*"
```

Comment:
@And012 @sapryniukt why this is needed?

Reply:
`node_modules/@ember/render-modifiers/package.json`
"ember-source": "^3.8 || 4" here is defined likes so.
Cause of this in local environment all scenarious that need ember-source above 4. 
CI was not happy and was collapsing.
<img width="983" alt="Screen Shot 2022-09-09 at 18 41 18" src="https://user-images.githubusercontent.com/62332020/189376271-400db34a-1c98-4b02-8efa-89f855aa17ab.png">

Reply:
during ember try:each it is added but I think I could remove it also